### PR TITLE
Polyhedron demo: add polylines/point sets, on the fly

### PR DESCRIPTION
**Pull request by maxGimeno**
This creates a Dialog to add a polyline by entering coordinates, and another one to do the same with a point set.

### CODE REVIEW

**Comment by sloriot** [comment]
I tested it locally and it works great.

**Comment by lrineau** [comment]
I do not like that the `MainWindow` class was changed:
  - with this PR `MainWindow` uses `Scene_polylines_item` directly,
  - the PR also adds an action to the File menu.

The demo is modular for good reasons. The feature should be in a plugin, and not in `MainWindow` directly. If you want to avoid add yet another plugin, it could be added to the `off_plugin`. Plugins can add actions to menus that are not the Operations menu. See the `init()` method of `demo/Polyhedron/Io_implicit_function_plugin.cpp`.

**Comment by maxGimeno** [comment]
I moved the features in polylines_io_plugin and xyz_plugin

File: Polyhedron/demo/Polyhedron/MainWindow.ui [hunk 1]
```diff
@@ -633,6 +632,16 @@
...
+   <property name="text">
+    <string>Add polylines</string>
+   </property>
+  </action>
+  <action name="actionAdd_point_set">
+   <property name="text">
+    <string>Add point set</string>
+   </property>
+  </action>
  </widget>
```
Comment (lrineau) [inline Polyhedron/demo/Polyhedron/MainWindow.ui]: `MainWindow.ui` is still modified by the PR.

**Comment by sloriot** [comment]
Merged in CGAL-4.8-Ic-40

**Comment by sloriot** [comment]
All warnings in the demo are not related to this branch.

**Comment by lrineau** [comment]
This PR is merged. 

@maxGimeno:  Please update https://cgal.geometryfactory.com/CGAL/Members/wiki/Staging_Area/Polyhedron_demo_enhancement

**Pull request merged**